pnpm-lock.yaml

```diff
@@ -13306,6 +13320,14 @@ snapshots:
 
   motion-utils@11.14.3: {}
 
+  motion@11.15.0(react-dom@18.3.1(react@18.3.1))(react@18.3.1):
+    dependencies:
+      framer-motion: 11.15.0(react-dom@18.3.1(react@18.3.1))(react@18.3.1)
```

Comment:
Maybe I'm not reading this correctly, but why does `motion@11.15.0` still have a dependency of framer-motion 11.15.0?

Reply:
My hunch was correct, looks like it is just a re-export of `framer-motion` at the moment.

https://github.com/motiondivision/motion/blob/ecd97f7dce8954be300aa73ab6a96208437941c5/packages/motion/src/react.ts#L3

Reply:
Thanks for checking.